### ISSUE TITLE
Duplicate Entries when adding profile type in register form

### ISSUE THREAD

**Issue by AmineBAIK1** [buddyboss/buddyboss-platform#1532]
Steps to reproduce : 
- Activate Registration 
- Add some profiles type 
- Add profile type select in register form 
![Capture d’écran de 2020-09-22 02-09-47](https://user-images.githubusercontent.com/46829598/93903140-9891c380-fcf8-11ea-899d-9eb2ca1bb5b0.png)
![Capture d’écran de 2020-09-22 02-09-49](https://user-images.githubusercontent.com/46829598/93903105-92034c00-fcf8-11ea-8815-f3c4989da8ae.png)

**Comment by sergiolucianojr** [comment]
I added the merged code directly in the bp-x-profile php code and the error persists.

Is this issue really fixed?

**Comment by dipskakadiya** [comment]
@sergiolucianojr Thanks for the reported issue. 

In our internal testing, we found it's a conflict with the profile display name setting. This is fixed now and working perfectly. We plan to release this fix soon. 

Meanwhile, you can put below code instead of https://github.com/buddyboss/buddyboss-platform/pull/1523/files change.
```php
if ( ! empty( $fields_id ) ) {
		if ( empty( $args['exclude_fields'] ) ) {
			$args['exclude_fields'] = [];
		}
		$args['exclude_fields'] = array_merge( wp_parse_id_list( $args['exclude_fields'] ), $fields_id );
	}
```
Let me know if anything does not work for you. If possible please creating a support ticket with your site credential so we can debug further.

**Comment by sergiolucianojr** [comment]
Yeaaaah! Tks for the solution. @dipskakadiya 

I just changed the code for this one and now it seems to work perfectly. <3

I will do some more tests and, if it is not working, I will back to you here.

**Comment by sourabhmatolia** [comment]
Hello @AmineBAIK1  @sergiolucianojr 

Issue created above is released in BuddyBoss Platform v1.5.2

Please find release notes here:
https://www.buddyboss.com/resources/buddyboss-platform-releases/1-5-2/

Please update the plugin to see the latest changes.

Thank you.
